libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/types.ts

```diff
@@ -284,3 +352,12 @@ export interface ISharedObjectUnderLenientControlProps {
     isLeniencyControlSupported: boolean;
     onUnderLenientControlChange: (isUnderLenientControl: boolean) => void;
 }
+
+/**
+ * @internal
+ */
+export interface IPermissionTypeItem {
```

Comment:
Maybe `IGranularPermissionTypeItem` would be better name.

Reply:
Renamed.